packages/shared/lib/core/stronghold/utils/getDefaultStrongholdName.ts

```diff
@@ -0,0 +1,8 @@
+export function getDefaultStrongholdName(): string {
+    // Match https://github.com/iotaledger/wallet.rs/blob/ffbeaa3466b44f79dd5f87e14ed1bdc4846d9e85/src/account_manager.rs#L1428
+    // Trim milliseconds and replace colons with dashes
+    const tzoffset = new Date().getTimezoneOffset() * 60000 // offset in milliseconds
```

Comment:
camelCase?

Reply:
Also the `60000` needs to be with our time constants